### PR TITLE
app list view and app detail added

### PR DESCRIPTION
**Pull request by adsworth**

### CODE REVIEW

File: nextcloudappstore/core/templates/home.html [hunk 1]
```diff
@@ -33,7 +24,7 @@
 {% for app in recommended_apps %}
   <div class="col-sm-4">
     <div class="well">
-      <h3><a href="http://getbootstrap.com">{{ app.name }}</a></h3>
+      <h3><a href="{%url 'app_detail' id=app.id %}">{{ app.name }}</a></h3>
```
Comment (BernhardPosselt) [inline nextcloudappstore/core/templates/home.html]: do we go for _ or - for separating words?

File: nextcloudappstore/core/views.py [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+
+class HomeView(CategoryContextMixin,
+               RecommendedAppsContextMixin, TemplateView):
     template_name = "home.html"
+
+
+class AppDetailView(CategoryContextMixin, DetailView):
+    model = App
+    template_name = 'app/detail.html'
+    slug_field = 'id'
```
Comment (BernhardPosselt) [inline nextcloudappstore/core/views.py]: I think you can name the parameter in the url slug and it just works :)

**Comment by BernhardPosselt** [comment]
Well, whatever looks good :)

As for separating words in URLs, would you go for - or _?

File: nextcloudappstore/core/views.py (hunk 2 above)
Comment (adsworth) [inline nextcloudappstore/core/views.py]: No, it doesn't. I tried that and when getting DetailView complains that it needs either a pk or slug kwarg.

**Pull request merged**